### PR TITLE
Switch flag to kingpin.

### PR DESCRIPTION
**Pull request by alokmenghrajani**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -30,43 +28,33 @@ import (
...
-	}
-
-	flag.Parse()
-	if flag.NArg() != 2 {
-		Usage()
-		os.Exit(1)
-	}
-
-	serverURL, err := url.Parse(flag.Args()[0])
-	if err != nil {
```
Comment (alokmenghrajani) [inline main.go]: do we care about `url.Parse`? kingpin's URL() doesn't seem to validate anything?

Comment (csstaub) [inline main.go]: It's fine, it'll just crash if it's invalid anyway.

**Comment by csstaub** [comment]
Nice. LGTM.

File: main.go (hunk 1 above)
Comment (mcpherrinm) [inline main.go]: Why do we even take a URL, instead of just host:port or host --port or something?

Comment (csstaub) [inline main.go]: Well, why not? makes it 1 argument instead of 2. seems good to me.

Comment (mcpherrinm) [inline main.go]: Well, I forget the https:// all the time, and we had awkwardness in the past with trailing slashes (though I think we fixed that).  localhost:4444 seems to do just as well as https://localhost:4444/

**Comment by mcpherrinm** [comment]
:shipit:

**Pull request merged**